manifests/machineosbuilder/deployment.yaml

```diff
@@ -0,0 +1 @@
+//TODO (dkhater) implement me!
```

Comment:
Let's actually populate this file with a basic deployment. You can use the `manifests/machineconfigcontroller/deployment.yaml` as a basis for this. I took a closer look at how the MCO applies these manifests and it won't just find the file and magically apply it (which is good). That said, we should at least lay the groundwork for that.

Reply:
Once you've done that, let's add the skeleton code to the MCO. To do that, you can use [this code](https://github.com/openshift/machine-config-operator/blob/c748a0f4da83aa58bb63b07da97d0f014ec6cfdd/pkg/operator/sync.go#L631-L668) as an example and create a `syncMachineOSBuilder()` method in that file. Just doing that will not cause these templates to be rendered and applied (which we don't want right now). Instead, we'll wire that up at a later date once we've figured out how the opt-in switch works.